imagetest/test_suites/oslogin/oslogin_test.go

```diff
@@ -96,3 +103,69 @@ func TestOsLoginDisabled(t *testing.T) {
 		t.Errorf("OS Login PAM module wrongly included in pam.d/sshd when disabled.")
 	}
 }
+
+func TestGetentPasswdOsloginUser(t *testing.T) {
+	cmd := exec.Command("/usr/bin/getent", "passwd", TEST_USERNAME)
+	out, err := cmd.Output()
+	if err != nil {
+		t.Fatalf("getent command failed %v", err)
+	}
+	if !strings.Contains(string(out), TEST_USER_ENTRY) {
+		t.Errorf("getent passwd output does not contain %s", TEST_USER_ENTRY)
+	}
+}
+
+func TestGetentPasswdAllUsers(t *testing.T) {
+	cmd := exec.Command("/usr/bin/getent", "passwd")
+	out, err := cmd.Output()
+	if err != nil {
+		t.Fatalf("getent command failed %v", err)
+	}
+	if !strings.Contains(string(out), "root:x:0:0:root:/root:") {
+		t.Errorf("getent passwd output does not contain user root")
+	}
+	if !strings.Contains(string(out), "nobody:x:") {
+		t.Errorf("getent passwd output does not contain user nobody")
+	}
+	if !strings.Contains(string(out), TEST_USER_ENTRY) {
+		t.Errorf("getent passwd output does not contain %s", TEST_USER_ENTRY)
+	}
+}
+
+func TestGetentPasswdOsloginUID(t *testing.T) {
+	cmd := exec.Command("/usr/bin/getent", "passwd", TEST_UID)
+	out, err := cmd.Output()
+	if err != nil {
+		t.Fatalf("getent command failed %v", err)
+	}
+	if !strings.Contains(string(out), TEST_USER_ENTRY) {
+		t.Errorf("getent passwd output does not contain %s", TEST_USER_ENTRY)
+	}
+	cmd = exec.Command("/usr/bin/getent", "passwd", TEST_USERNAME)
+	out, err = cmd.Output()
+	if err != nil {
+		t.Fatalf("getent command failed %v", err)
+	}
+	if !strings.Contains(string(out), TEST_USER_ENTRY) {
+		t.Errorf("getent passwd output does not contain %s", TEST_USER_ENTRY)
+	}
```

Comment:
How is this different to `TestGetentPasswdOsloginUser` and why is asserting against `TEST_USER_ENTRY` when `TEST_USERNAME` is provided?

Reply:
the output of the command should be what's in `TEST_USER_ENTRY`, exactly. compared to the less specific `TEST_USERNAME` which will only be part of the output

Reply:
So, in discussions with Liam before the PR, it was noted that we should test both that asking for username returns the correct UID and asking for UID returns the correct username. TEST_USER_ENTRY contains both the username and the UID. However, when I added the test to make sure that username returns UID, I didn't realize that TestGetentPassweOsloginUser already tests this. I'll remove this one and just let the previous test cover it.